cluster-autoscaler/cloudprovider/digitalocean/README.md

```diff
@@ -0,0 +1,58 @@
+# Cluster Autoscaler for DigitalOcean
+
+The cluster autoscaler for DigitalOcean scales worker nodes within any
+specified DigitalOcean Kubernetes cluster's node pool. This is part of the DOKS
+offering which can be enabled/disable dynamically for an existing cluster.
+
+# Configuration
+
+The `cluster-autoscaler` dynamically runs based on tags associated with node
+pools. These are the current valid tags:
```

Comment:
This is completely different from how you run CA on different providers. Any particular reason for this design?

Reply:
This let us dynamically enable/disable the cluster-autoscaler and make it by default as `opt-out`. So to opt-in you need to add those tags. Because our UI is not ready yet, this will allow  some customers to enable it by adding the tags themself. Also this doesn't require us fiddle with CA flag arguments. It's easier to maintain and ship cluster-autoscaler and requires zero to minimum interaction once it's deployed. Is there something you believe is wrong in this approach?

Reply:
Oh, I didn't realize you're planning to host CA pod (or at least start it automatically). I don't think there is anything inherently wrong with your approach, it's just inconsistent with other providers.
Given that autodiscovery already works differently for each provider I don't think it's a blocker for merging this either. Just a suboptimal experience for someone moving between clouds / running multicloud / etc.